### DISCUSSION TITLE
add swipe pager features.. in.. Tabmenu

### DISCUSSION THREAD

**Discussion by ecomdeveloperbd** [inspireui/support#5659] [New Feature/Ideas Requests]
when multiple dynamic 5-10+ page include single Tabmenu ..... every tab menu option click then change dynamic page....

i would suggest please. it's Tabmenu option add swipe option.... just swipe change the dynamic page from Tabmenu.

example: swipe option... show the video
Link: https://streamable.com/8glky6

**Comment by osman-kocak** [comment]
I need thıs
